recipes/libcf/meta.yaml

```diff
@@ -0,0 +1,56 @@
+package:
+{% set name = "libcf" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}-beta11.tar.gz
+  url: http://uvcdat.llnl.gov/cdat/resources/{{ name }}-{{ version }}-beta11.tar.gz
+  sha256: e40d3826ca2b47538b075f69f716798e255fe396fd3ae68e8b051be57a951bb4
+
+build:
+  number: 0
+  skip: True  # [win] 
+
+requirements:
+  build:
+    - hdf5
+    - libnetcdf 4.3.3.1
```

Comment:
Do you need this strict pinning for `libnetcdf`? Please see conda-forge pinning docs at:
https://github.com/conda-forge/staged-recipes/wiki/Pinned-dependencies